larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -503,15 +512,29 @@ TEST_P(BConv2DOpTest, SimpleTest) {
       {TensorType_FLOAT32,
        {filter_count, filter_height, filter_width, input_depth}},  // filter
       {TensorType_FLOAT32, {}},                                    // output
-      stride_width, stride_height, builtin_padding, ActivationFunctionType_NONE,
+      stride_width, stride_height, builtin_padding, activation,
       dilation_width_factor, dilation_height_factor, num_threads);
 
   m_builtin.SetInput(padded_input_data);
   m_builtin.SetFilter(filters_data);
   m_builtin.SetBias(bias_data);
   m_builtin.Invoke();
-
   auto expected_array = m_builtin.GetOutput();
+
+  // Apply the post multiply and add to the tflite model
```

Comment:
can't you replace this with a TF Op?

Reply:
Yes but that would be *a lot* of extra code, I think it would make things a lot more unreadable. It would mean all these extra classes at the top of the file, with `MulOp` and `AddOp` and so on. Then we would have all the code to setup the op, set input tensors, read output tensors etc etc. In my opinion this is a lot cleaner.

Reply:
ok, agreed 👍